tools/uart_hw_test/README.md

```diff
@@ -40,8 +40,21 @@ It is MANDATORY to first run the uart_hw_test tash command and enter the port nu
 After this run the ./uart_test.sh test app on linux and enter the port number.
 If this sequence is not followed, then the result is not proper.
 
+### How to USE for windows
+1. Find the UART port to be tested. For example, on the RTK8721 board, we will perform uart test for the uart1.
+2. Connect the uart (Rx, Tx, Gnd) lines from the test device to the Windows PC using Serial-USB converter.
+3. Enable CONFIG_UART_HW_TEST config. Build and download the binary.
+4. After boot up, run the uart_hw_test command at the tash prompt (find the appropiate console for windows to get tash prompt).
+5. Input the appropriate port number .
+6. Now compile the uart_test_windows.c file in windows using "gcc -o test uart_test_windows.c" and it will create test.exe executable file.
+7. Now run "test" in windows and input the appropriate COM port number(find the required COM port number from device manager).
```

Comment:
The user may use any naming for the generated object file.
Hence this text should be described as "For Example:"

Also, using the name "test" for the executable file may not be the best option.